### ISSUE TITLE
Add "change test result" to bulk edit action on samples directory

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#3885]
### Situation Description
There are situations where several cases with negative results are left pending on the samples directory, its therefore pertinent to have a bulk action where an admin can change the result in bulk on the samples directory @bernardsilenou

**Comment by Iheanacho2027** [comment]
Pls kindly give this urgent consideration @bernardsilenou. Thanks

**Comment by bernardsilenou** [comment]
@Iheanacho2027 I think it would be good to have a meeting to refine this issue and other similar once related to bulk sample results import and editing.
- We can assign a "final laboratory result" to a sample by creating a "new test result". Each test result may have a different information thus making it complex to have a bulk action. Maybe importation of results can be an options but this would need the uuid or "field sample id" to be attached to each result.
- Please can you take time to explain more about the needs 
- Send doodle invite by email pls, me and @kwa20 would vote as one.

**Comment by Iheanacho2027** [comment]
> * We can assign a "final laboratory result" to a sample by creating a "new test result". Each test result may have a different information thus making it complex to have a bulk action. Maybe importation of results can be an options but this would need the uuid or "field sample id" to be attached to each result.
Yes this is very true. I agree with you. The bulk action on changing the laboratory result would be for the "final laboratory result" variable. For "new test result", information can be ignored and later on handled by samples import. 

> * Send doodle invite by email pls, me and @kwa20 would vote as one.
What time will you be available?

**Comment by bernardsilenou** [comment]
@Iheanacho2027 
* The point here is that the "final lab result" of the sample is not something like the case classification that you can manually change and we do not want to make it open for changes because it depends on one of the lab results of the sample. We can discuss about this more when we call.
* We cant arrange the meeting here, please use https://doodle.com/
* You can already discuss with Chinedu and Ferdinand to have possible dates before proposing. I would then vote the HZI team

**Comment by Iheanacho2027** [comment]
Ok, let's see how far importation of samples results will solve this. I believe it should. If it does, then the ticket can be closed
